packages/runtime/plugin-runtime/src/core/server/requestHandler.ts

```diff
@@ -25,7 +26,7 @@ export type HandleRequestOptions = Exclude<
 
 export type HandleRequest = (
   request: Request,
-  serverRoot: React.ComponentType, // App, routes,
+  ServerRoot: React.ReactElement, // App, routes,
```

Comment:
the name for `ReactElement` type is better use lowercase, serverRoot is good

Reply:
Done